Oqtane.Shared/Shared/Constants.cs

```diff
@@ -72,7 +72,7 @@ public class Constants {
         };
         public static readonly string[] InvalidFileNameEndingChars = { ".", " " };
 
-        public static readonly string SatelliteAssemblyExtension = ".resources.dll";
+        public static readonly string SatelliteAssemblyExtension = "*.resources.dll";
```

Comment:
I presume this is an extension which is fine, if there's a bug in search we could fix in the place where search occurs

Reply:
It occurs in `static class OqtaneServiceCollectionExtensions` 
on `static void LoadSatelliteAssemblies(string[] supportedCultures)`

The code in foreach would **never** execute without a wildcard
`foreach (var assemblyFile in assembliesFolder.EnumerateFiles(Constants.SatelliteAssemblyExtension))`

Reply:
I created #1544 which don't break the satellite extension